ibm_catalog.json

```diff
@@ -0,0 +1,98 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-observability-instances",
+        "label": "Deployment of IBM Cloud Observability Instances",
```

Comment:
supports agents too

Reply:
yes leave it in separate PR, but don't mention instances here - the offering will eventually do more than that so just maybe say Observability resources ?